internal_deployment/composer_env.tf

```diff
@@ -0,0 +1,58 @@
+resource "google_composer_environment" "cloud_ml_auto_solutions_env" {
+  provider = google-beta
+  name     = var.env_name
+  region = var.region
+
+  config {
```

Comment:
Instead of having a separate copy for those configs ([source](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/deployment/cloud_composer_template.tf#L131)), could we move it to a common file, and reuse for both developer and prod envs?

Reply:
Sounds good - I moved the cloud composer resource into its own module, and am now referencing that in both ./deployment and ./deployment/deployment_composer_env

Reply:
SG! Thanks!